XCalendar.Maui/Converters/LocalizeDayOfWeekAndCharLimitConverter.cs

```diff
@@ -8,25 +7,14 @@ public class LocalizeDayOfWeekAndCharLimitConverter : IValueConverter
     {
         public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
         {
-            if (value is not DayOfWeek dayOfWeek)
+            if (!(value is DayOfWeek dayOfWeek))
             {
                 return string.Empty;
             }
 
-            string resourceName = $"{dayOfWeek}";
-
-            string dayName = XCalendarResource.ResourceManager.GetString(resourceName, culture);
-
-            if (string.IsNullOrWhiteSpace(dayName))
-            {
-                dayName = dayOfWeek.ToString().Substring(0, 3);
-            }
-            else
-            {
-                dayName = dayName.TruncateStringToMaxLength(parameter);
-            }
-
-            return dayName;
+            return dayOfWeek.LocalizeDayOfWeek(culture)
+                .TruncateStringToMaxLength(parameter)
+                .UppercaseFirst();
```

Comment:
The uppercasing of the first letter might not be needed. I think the string returned from "DateTimeFormatInfo.GetDayName" always start with an upper-cased letter

Reply:
In English, yes, but not in Spanish. I don't know why, but we cannot ensure behaviors in other cultures. So, to ensure, we must capitalize